{"uuid": "95b40163-932b-4396-92ce-5e74669fbe86", "children": ["e118390b-e838-48a5-af3c-67fea19a1c93"], "befores": [{"name": "capture_screenshot_on_failure", "status": "passed", "start": 1741860076535, "stop": 1741860076535}], "afters": [{"name": "capture_screenshot_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\rutwik\\miniconda3\\envs\\automation\\lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\rutwik\\Automation\\conftest.py\", line 105, in capture_screenshot_on_failure\n    if request.node.rep_call.failed:\n"}, "start": 1741860076857, "stop": 1741860076862}], "start": 1741860076535, "stop": 1741860076862}